test/support/conn_case.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Redbird.ConnCase do
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      use Plug.Test
+      import Redbird.ConnCase
+    end
+  end
+
+  @default_opts [store: :redis, key: "_session_key"]
+
+  def signed_conn do
+    :get |> Plug.Test.conn("/") |> sign_conn()
+  end
+
+  def sign_conn(conn, options \\ []) do
+    sign_conn_with(conn, generate_secret(), options)
+  end
+
+  def sign_conn_with(conn, secret, opts \\ []) do
+    conn.secret_key_base
+    |> put_in(secret)
+    |> Plug.Session.call(sign_plug(opts))
+    |> Plug.Conn.fetch_session()
+  end
+
+  def sign_plug(options) do
```

Comment:
Can this function be private? I don't see it used anywhere else other than in `sign_conn_with/3`.

Reply:
Good call. Done.